core/package.json

```diff
@@ -59,7 +59,8 @@
     },
     "testMatch": [
       "**/__test__/*.(ts|tsx|js)"
-    ]
+    ],
+    "testURL": "http://localhost/"
```

Comment:
FYI I had to add that to make the tests work as `jest` throws `SecurityError: localStorage is not available for opaque origins` otherwise, the reason being that `testURL` defaults to `about:blank` in [v0.22](https://jestjs.io/docs/en/22.x/configuration#testurl-string) whereas it has been fixed to `http://localhost/` in the [latest version](https://jestjs.io/docs/en/configuration#testurl-string).

This can be removed once `jest` has been upgraded, but this was beyond the scope of the PR.